src/ontology/mondo-edit.obo

```diff
@@ -277848,6 +277881,7 @@ property_value: confidence "0.5294117647058822" xsd:double
 [Term]
 id: MONDO:0014141
 name: muscular dystrophy-dystroglycanopathy (congenital with intellectual disability), type B14
+synonym: "congenital muscular dystrophy-GMPPB related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/